src/view/uniprotkb/components/ProteinNamesView.tsx

```diff
@@ -81,6 +81,22 @@ const ProteinDescriptionView: React.FC<{
   );
 };
 
+export const ECNumbersView: FC<{
+  ecNumbers: ValueWithEvidence[];
+  isCompact?: boolean;
+}> = ({ ecNumbers, isCompact = false }) => (
+  <Fragment>
+    {ecNumbers.map(
+      (ecNumber): JSX.Element =>
+        isCompact ? (
+          <Fragment key={v1()}>{ecNumber.value}</Fragment>
+        ) : (
+          <NameWithEvidence data={ecNumber} key={v1()} />
```

Comment:
No action needed now but suggesting it as a larger conversation: do want to add uuid when we convert the data (across the whole front-end codebase)? This way the ids will be static and the tree diff'ing can work as normal.

Reply:
Most definitely as this is currently useless. Will make a Jira